BlazorWasmGraphQL/Client/wwwroot/index.html

```diff
@@ -10,7 +10,8 @@
     <link href="css/app.css" rel="stylesheet" />
     <link href="BlazorWasmGraphQL.Client.styles.css" rel="stylesheet" />
     <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" />
-    <link href="_content/Blazored.Toast/blazored-toast.min.css" rel="stylesheet" />
+    <link href="_content/Syncfusion.Blazor.Themes/bootstrap5.css" rel="stylesheet" />
+    <script src="_content/Syncfusion.Blazor.Core/scripts/syncfusion-blazor.min.js" type="text/javascript"></script>
```

Comment:
Modify this to individual package reference (Syncfusion.Blazor.Notifications)

Reply:
Already I have used the Syncfusion.Blazor.Notifications